components/brave_rewards/browser/rewards_service_observer.h

```diff
@@ -61,6 +64,13 @@ class RewardsServiceObserver : public base::CheckedObserver {
   virtual void OnPublisherListNormalized(
       RewardsService* rewards_service,
       brave_rewards::ContentSiteList list) {};
+  virtual void OnAdsNotificationsData(brave_rewards::RewardsService* rewards_service,
```

Comment:
I think it was just a mistake listing this twice?

Reply:
Strange in my local copy I do not see this dupe?